src/profile-logic/symbol-store.js

```diff
@@ -198,9 +198,8 @@ export class SymbolStore {
   }
 
   // Store a symbol table in the database. This is only used for symbol tables
-  // and not for partial symbol results. Symbol tables are generated by the
-  // geckoProfiler WebExtension API, so these are symbol tables we get from the
-  // add-on.
+  // and not for partial symbol results. Symbol tables are obtained from the
+  // browser via the geckoProfiler object which is defined in a frame script.
```

Comment:
optional nit: would it be a good idea to put a pointer to the frame script on searchfox or hg.mozilla.org ?

Reply:
Good idea.